### PR TITLE
ES: EP5 Cap2-3

### PR DESCRIPTION
**Pull request by NozomiRem**

### CODE REVIEW

File: Misc/Concert/bd_010030.csv [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+bd_010030_0100#0,"""Hacia la gran aventura<br><br><blu>Biigusukeru no adobencha<c>"""
+bd_010030_0110#0,"""¡Vamos a despegar!<br><br><blu>Min na de tobiko mou yo<c>"""
+bd_010030_0120#0,"""Los desconocidos Flashing Skies serán nuestro escenario<br><br><blu>Flashing sky michi o himeta stage<c>"""
+bd_010030_0130#0,"""Un misterioso destino lleno de brillantes joyas<br><br><blu>Hoseki o maki chirashita shinpi no destination<c>"""
+bd_010030_0140#0,"""Las proyecciones del polvo de estrellas Merry-go-round<br><br><blu>Horoguramu no keshiki ga mawaru Stardust Merry-Go-Round<c>"""
+bd_010030_0150#0,"""Hacia lo desconocido<br><br><blu>Mi taiken no uzu we<c>"""
+bd_010030_0160#0,"""Sigue adelante sin dudas<br><br><blu>Mayoi toma dou toki mo ushiro o furimui tari shinaide<c>"""
+bd_010030_0170#0,"""Siempre el mañana brilla ante mis ojos<br><br><blu>Itsumo me no mae ni ka ga ya ku ashita wa hirogaru no<c>"""
+bd_010030_0180#0,"""La galaxia de estrellas parpadeantes ilumina nuestros sueños"<br><br><blu>Ginga o megure twinkle star yume o terashi da se<c>""
```
Comment (alama) [inline Misc/Concert/bd_010030.csv]: Missing a `"` at the end of this line

Comment (NozomiRem) [reply Misc/Concert/bd_010030.csv]: repair what the  " and the error continues

Comment (NozomiRem) [reply Misc/Concert/bd_010030.csv]: linea 23 faill

File: Misc/Concert/bd_010030.csv [hunk 2]
```diff
@@ -20,7 +20,7 @@ bd_010030_0140#0,"""Las proyecciones del polvo de estrellas Merry-go-round<br><b
 bd_010030_0150#0,"""Hacia lo desconocido<br><br><blu>Mi taiken no uzu we<c>"""
 bd_010030_0160#0,"""Sigue adelante sin dudas<br><br><blu>Mayoi toma dou toki mo ushiro o furimui tari shinaide<c>"""
 bd_010030_0170#0,"""Siempre el mañana brilla ante mis ojos<br><br><blu>Itsumo me no mae ni ka ga ya ku ashita wa hirogaru no<c>"""
-bd_010030_0180#0,"""La galaxia de estrellas parpadeantes ilumina nuestros sueños"<br><br><blu>Ginga o megure twinkle star yume o terashi da se<c>""
+bd_010030_0180#0,"""La galaxia de estrellas parpadeantes ilumina nuestros sueños"<br><br><blu>Ginga o megure twinkle star yume o terashi da se<c>"""
```
Comment (alama) [inline Misc/Concert/bd_010030.csv]: replace `"` with `\""`

**Pull request merged**